TimberAPI/AssetLoaderSystem/ResourceAssetPatch/TimberApiResourceAssetLoader.cs

```diff
@@ -47,22 +61,28 @@ public TimberApiResourceAssetLoader(IAssetLoader assetLoader)
             return Resources.LoadAll<T>(path);
         }
 
-        private static void FixMaterialShader(GameObject obj, Shader shader)
+        private static void FixMaterialShader(GameObject obj)
         {
             var meshRenderer = obj.GetComponent<MeshRenderer>();
             if (meshRenderer)
             {
                 foreach (Material mat in meshRenderer.materials)
                 {
-                    mat.shader = shader;
+                    
+                    TimberAPIPlugin.Log.LogFatal(mat.renderQueue);
+                    var matRenderQueue = mat.renderQueue; 
```

Comment:
We get the renderQueue here, do nothing with it, and 2 lines later we assign it back? Should be removed

Reply:
This is because if a shader is changed, the renderqueue from the material is also changed, but this will make it so it is rendered incorrectly. So I save the queue and than overwrite the queue again. For some reason the shader that gets loaded, doesn't have the correct renderqueue.

Reply:
🤔 well that's interesting, good to know

Reply:
I know this is already merged, but you might add a comment on this. I generally find weird cases like this to be good places for comments.